server/service/src/sync/test/test_data/vvm_status.rs

```diff
@@ -0,0 +1,70 @@
+use repository::vvm_status_row::{VVMStatusRow, VVMStatusRowDelete};
+
+use crate::sync::test::TestSyncIncomingRecord;
+
+const TABLE_NAME: &str = "vvm_status";
+
+const VVM_STATUS_1: (&str, &str) = (
+    "VVM_STATUS_1",
+    r#"{
+        "ID":"VVM_STATUS_1",
+        "code":"2",
+        "description":"TEST DESCRIPTION 1",
+        "is_active":true,
+        "level":1,
+        "unusable":false,
+        "option_id":"TEST OPTION ID 1"
```

Comment:
Lets leave option_id out for now...

Ideally it should be linked to the `reason_option` table I think, and should really have a foreign key connection in the database. I think we can fill that in later when we have a real example?
```suggestion
        "option_id":""
```

Reply:
Why did you resolve this one? Did you feel it's not needed to change?

Reply:
I believe I committed it, and then it automatically resolved, but I take it that that is different from what you are seeing?

Reply:
Ah, and I actually think I reversed the change in the latest commit, as it was causing a test to fail.